docs/java-rest/high-level/indices/shrink_index.asciidoc

```diff
@@ -47,12 +47,21 @@ returns a response, as an `int`
 <2> The number of active shard copies to wait for before the shrink index API
 returns a response, as an `ActiveShardCount`
 
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests-file}[{api}-request-copySettings]
+--------------------------------------------------
+<1> Use `Boolean.TRUE` to copy the settings from the source index to the target
```

Comment:
Use `true`

Reply:
Addressed in de8ca1863c29ea222fe23b3366d207a4bdbcb4de